### PR TITLE
Handle checksums

### PR DESCRIPTION
**Pull request by ldko**
This adds checksum verification and manifest file generation. Also, there is now a commandline flag just to print out a list of URLs for the downloadable files.

### CODE REVIEW

File: tests/test_wasapi_client.py [hunk 1]
```diff
@@ -164,6 +168,83 @@ def test_populate_downloads_multi_page(self, mock_session):
         for field in ('locations', 'filename', 'checksums'):
             assert field in q_item
 
+    def test_populate_downloads_no_get_q(self, mock_session):
+        """Test downloads=False prevents get_q attribute existing."""
```
Comment (somexpert) [inline tests/test_wasapi_client.py]: `downloads` argument should be singular

File: tests/test_wasapi_client.py [hunk 2]
```diff
@@ -287,34 +454,28 @@ class TestDownloader:
...
         log_q = multiprocessing.Queue()
-
-        p = wc.Downloader(get_q, result_q, log_q)
-        p.start()
-        p.run()
-        p.join()
+        with patch('wasapi_client.verify_file', return_value=True):
+            p = wc.Downloader(get_q, result_q, log_q)
+            p.start()
+            p.run()
```
Comment (somexpert) [inline tests/test_wasapi_client.py]: Does p.run() actually need to be called? Isn't it enough to just call p.start()?

**Review by somexpert — COMMENTED** [review]
Looks good and all tests passed for me. One quick question: do you think it would be worth it to add an example of using the `--manifest` option to the README?

File: tests/test_wasapi_client.py (hunk 1 above)
Comment (ldko) [reply tests/test_wasapi_client.py]: you're right, good eye :wink:

File: tests/test_wasapi_client.py (hunk 2 above)
Comment (ldko) [reply tests/test_wasapi_client.py]: No it doesn't need to be called explicitly.

**Comment by ldko** [comment]
yeah, I think it would be worth adding --manifest (and --urls) example/info to the README. I am planning to add travis stuff and update the readme after that with more usage help. Thanks!

**Comment by somexpert** [comment]
I don't know how to change my review status from being a comment to an approval, so here ya go! :thumbsup:

**Pull request merged**